pkgs/top-level/all-packages.nix

```diff
@@ -2029,9 +2029,11 @@ with pkgs;
 
   android-backup-extractor = callPackage ../tools/backup/android-backup-extractor {};
 
-  android-tools = lowPrio (callPackage ../tools/misc/android-tools {
-    stdenv = if stdenv.targetPlatform.isAarch64 then gcc10Stdenv else stdenv;
-  });
+  android-tools = lowPrio (darwin.apple_sdk_11_0.callPackage ../tools/misc/android-tools
+    (lib.optionalAttrs (stdenv.targetPlatform.isAarch64 && stdenv.targetPlatform.isLinux) {
+      stdenv = gcc10Stdenv;
+    }));
+
```

Comment:
Can you drop the extraneous new line here?

(Also: was it intentional to not balance the alignment on that last line?)

Reply:
Done. For the formatting, that's what nixpkgs-fmt gave me...

Reply:
Huh, weird. Guess it doesn't matter too much, that file is a mess anyways.

*wishes for RFC 101*